scripts/lint.bash

```diff
@@ -5,5 +5,7 @@ shellcheck --shell=bash --external-sources \
     lib/* \
     scripts/*
 
+# shellcheck disable=SC2046
+# shellcheck disable=SC2156
```

Comment:
![image](https://github.com/asdf-community/asdf-plugin-manager/assets/61225/67476506-21e7-4d5d-9f84-860d27a502ab)

Reply:
Please fix them instead of disabling them.